clerk/users.go

```diff
@@ -249,6 +249,27 @@ func (s *UsersService) DisableMFA(userID string) (*UserDisableMFAResponse, error
 	return &response, nil
 }
 
+type OAuthProviderToken struct {
+	Object         string      `json:"object"`
+	Provider       string      `json:"provider"`
+	Token          string      `json:"token"`
+	PublicMetadata interface{} `json:"public_metadata"`
+	Label          string      `json:"label"`
+	Scopes         []string    `json:"scopes"`
```

Comment:
Turns out that the `scopes` property is available only OAuth 2.0 tokens. Oauth 1.0 tokens use a `token_secret` property.
```suggestion
        // Only set in OAuth 2.0 tokens
	Scopes []string    `json:"scopes,omitempty"`
	// Only set in OAuth 1.0 tokens
	TokenSecret string      `json:"token_secret,omitempty"`
```